psi4/driver/driver_nbody.py

```diff
@@ -163,18 +179,45 @@ def nbody_gufunc(func, method_string, **kwargs):
 
         If True returns the total data (energy/gradient/etc) of the system,
         otherwise returns interaction data.
+
+    :type levels: dict
+    :param levels: ``{1: 'ccsd(t)', 2: 'mp2', 'supersystem': 'scf'} || {1: 2, 2: 'ccsd(t)', 3: 'mp2'}
```

Comment:
Means extract the one-body energy from the two-body energy. If you say `{1: 'ccsd(t), 2: 'ccsd(t)}`, it will have redundant calculations for the one-body term.